stable/aws-load-balancer-controller/templates/webhook.yaml

```diff
@@ -36,7 +36,7 @@ webhooks:
     - key: app.kubernetes.io/name
       operator: NotIn
       values:
-      - aws-load-balancer-controller
+      - {{ include "aws-load-balancer-controller.fullname" . }}
```

Comment:
seems we are using `aws-load-balancer-controller.name`

Reply:
thanks. I will update